webdriver_manager/drivers/chrome.py

```diff
@@ -29,10 +30,22 @@ def get_os_type(self):
             return os_type
 
         if is_arch(os_type):
-            return f"{os_type.replace('_m1', '')}_m1"
+            return "mac_arm64"
```

Comment:
ok, but if it's arch that is not mac?

Reply:
previous if statement checks "none mac" case, and returns os_type as is.

But yeah, is_arch function is kind of misleading atm, i was even thinking to rename to "is_mac_arch" and add to inside function check if that's mac or not, cause it's always used after  is_mac check in current codebase

Reply:
Thanks for the fix!

I always wondered, why the function is named `is_arch` and it isn't `is_arm`? Isn't `is_arch` coming from is **arch**itecture (which can be any arch)?